package.json

```
@@ -271,6 +271,8 @@
       "typings"
     ],
     "nohoist": [
+      "@fluentui/docs",
+      "@fluentui/e2e",
```

This is broader than necessary and might have some unintended side effects due to introducing duplication of dependencies. I *think* this will work instead (you'll have to try it out).
```suggestion
      "**/@types/react-router-dom/@types/react-router",
```

---

nohoist docs https://classic.yarnpkg.com/blog/2018/02/15/nohoist/

---

Never mind, it's this issue again (I hate not being able to comment in package.json...): https://github.com/microsoft/fluentui/pull/21780#discussion_r812506315

So the actual solution is removing these two lines
```suggestion
```

Then removing the `@storybook/addons/` qualifier from these two later `nohoist` lines
https://github.com/microsoft/fluentui/blob/87f555dfcbe007e701f80b56e3c69c50141e5466/package.json#L287-L288